common/src/main/java/io/netty/util/internal/UnpaddedInternalThreadLocalMap.java

```diff
@@ -32,7 +32,7 @@
  */
 class UnpaddedInternalThreadLocalMap {
 
-    static ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap;
+    static final ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap = new ThreadLocal<InternalThreadLocalMap>();
```

Comment:
@windie I think it makes sense to move instance creation in `slowGet()` into the `ThreadLocal` then.

```java
    static final ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap = new ThreadLocal<InternalThreadLocalMap>() {
        @Override
        public InternalThreadLocalMap initialValue() {
            return new InternalThreadLocalMap();
        }
    }
```